editor/src/clj/editor/field_expression.clj

```diff
@@ -33,21 +34,33 @@
       (math/round-with-precision (op a b) precision))
     (parse-fn (parsable-number-format text))))
 
-(defn to-int [s]
+(defn- to-double-safe [s]
+  (try
+    (-> (ExpressionBuilder. s) .build .evaluate)
```

Comment:
I think we still want to call `math/round-with-precision` on the resulting value after evaluating the expression. 🤔

This gets rid of the annoying number of decimals you tend to get from the resulting values of double arithmetic.

Reply:
Sure, just added roud-with-precision